docs/architecture/architecture.md

```diff
@@ -0,0 +1,671 @@
+# Clear Containers Architecture
+
+* [Overview](#overview)
+    * [Hypervisor](#hypervisor)
+    * [Agent](#agent)
+    * [Runtime](#runtime)
+        * [Configuration](#configuration)
+        * [Significant commands](#significant-commands)
+            * [create](#create)
+            * [start](#start)
+            * [exec](#exec)
+            * [kill](#kill)
+            * [delete](#delete)
+    * [Proxy](#proxy)
+    * [Shim](#shim)
+    * [Networking](#networking)
+    * [Storage](#storage)
+* [Appendices](#appendices)
+    * [DAX](#dax)
+    * [Previous Releases](#previous-releases)
+    * [Resources](#resources)
+
+## Overview
+
+This is an architectural overview of Clear Containers, based on the 3.0 release.
+
+The [Clear Containers runtime (cc-runtime)](https://github.com/clearcontainers/runtime)
+is compatible with the [OCI](https://github.com/opencontainers) [runtime specification](https://github.com/opencontainers/runtime-spec)
```

Comment:
Hi @iphutch - does OCI need an asterisk here?

Reply:
Yes, we will need an asterisk here. Good catch.